src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SourcelinkTests.cs

```diff
@@ -14,6 +14,15 @@
 
 namespace Microsoft.DotNet.SourceBuild.SmokeTests;
 
+/// <summary>
+/// Separate test collection for Sourcelink tests. This is needed due to intra-test parallelization,
+/// which can cause less CPU time to be allocated to other tests.
+/// This would make other tests run too long and fail due to timeouts.
+/// </summary>
+[CollectionDefinition(nameof(SourcelinkTestCollectionDefinition), DisableParallelization = true)]
+public class SourcelinkTestCollectionDefinition { }
```

Comment:
What's the value of the Definition suffix here?  In the [xunit documentation](https://xunit.net/docs/shared-context) they don't seem to use this pattern.

Reply:
Right - I did copy one of the examples of this usage. Will clean this.

Reply:
Fixed in https://github.com/dotnet/installer/pull/17477/commits/fd4678ef74fb8870f6344668c765b4da5543fc35